server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/cache/CassandraDumbBlobStoreCache.java

```diff
@@ -0,0 +1,119 @@
+package org.apache.james.blob.cassandra.cache;
```

Comment:
We are missing...

 - A configurable TTL
 - Use of ConsistencyLevel.ONE

Reply:
>     * A configurable TTL

I dont see any memtion it in ticket.So what the value default of it?